### ISSUE TITLE
Simple(?) feature request for webMAN MOD and a possible BUG

### ISSUE THREAD

**Issue by Edw590** [aldostools/webMAN-MOD#454]
Hi. Could you please add an option on webMAN to change the time and date to some specific one we'd choose (being the current one an option if possible - if too hard, forget that one we'd just write it ourselves) with a combo, please?

Would be really helpful in something I needed. I need to enter a game with date on 2018, but I then need to sign in and play online the game with the current date, or the game servers won't let me go. I'm not sure this would do it, but I guess it would, so would be amazing if you could do it in a future update and I'd appreciate it a lot.

Thank you in any case and Merry Christmas!

**Comment by Edw590** [comment]
After asking this, I was told we can do that with a syscall and that webMAN executes syscalls. So I tried, and it worked, which is amazing. Though, when I do it in game, with the option to execute a command with the custom combo R2+[], it freezes the screen and part of the PS3. I'm saying part because I pressed Admin control combo and the PS3 beeped. It didn't freeze completely.

Is this a bug? Was it supposed to work while in game? The game I'm talking about is GTA 5, BLES version if that helps in any way.

**EDIT:** Either through the computer with the URL or with the combo and the custom command. The PS3 will freeze. No problem out of the game.

### Possibly useful infos:
- SEN Enabler 6.2.7 installed with the plugin enabled, but disabled before starting the game;
- No other plugins enabled when running the game except webMAN MOD, latest version, 1.47.32;
- Command: syscall.ps3?146|0x5FE671DE|0. On the 1st parameter, tried with decimal form in both parameters, and also tried to put nanoseconds equivalent of the 1st parameter on the 2nd parameter (if that's how the syscall works - I got a bit confused with it, but works nevertheless). Always freezes. But the date seems to change even with the 2nd parameter to 0, so I left it like this because I thought it could be related to processing too many characters, so I removed the 2nd parameter --> didn't do anything, it still freezes...;
- MEGA USEFUL, the date is changed successfully on SingStar. But it doesn't change on GTA 5 and partially freezes instead, no idea why. Possibly it's a game which pushes too much the PS3 to the limits? Could you maybe reduce the processing power needed to execute at least this syscall, if that's even possible? (or all syscalls, which would be even better)

**Comment by aldostools** [comment]
If the syscall works with SingStar but not with GTAV, then it's a game issue.

You could try using the rebug edition instead of the full edition, in case the freeze is a memory issue.
Also you can try disabling the FTP and NET servers to save some memory.

**Comment by Edw590** [comment]
Rebug edition is more lite than Lite edition? (I didn't even remember Rebug Toolbox had that option there haha)

Also about those 2, I'll try and see if it works.

**EDIT:** Except webMAN doesn't come in Rebug Toolbox anymore since 4.81.2 hahaha. I'll go with the Lite version then.

**Comment by aldostools** [comment]
It is unrelated to Rebug Toolbox. The "Rebug-PS3MAPI" edition is the standard edition that gets installed when you hold down R1 during the installation.

The "full" edition is installed when you hold down L1 during the installation.

The "Lite" edition is the lightest version, but it doesn't support the web command "/syscall.ps3" among other features.
The next lightest version is "rebug" or "standard" edition. And the heaviest version is the "full" edition.

**Comment by Edw590** [comment]
Hmm. I only have 2 options, not 3. Either Full (L1) or Lite (X). Is that supposed to show 3? (I remember before we would have 3 options, but it was years ago, maybe 2 or 3 - that's when I remember, at least, as it might have stopped later than that) I'll try with R1 and see what happens, if anything happens, since I don't see the option on the installer name.

**EDIT:** Well, it certainly did something. The web page changed. Wonder why there's no option showing for R1 though. Weird. Bug? Or maybe meant to be like that. But I think it's installed, so cool, thanks!

**Comment by aldostools** [comment]
Not a bug. The "Rebug-PS3MAPI" is the default plugin. That's why R1 is not mentioned in the scrolling text.

**Comment by Edw590** [comment]
Nope. And I also tested something. The game has a mod menu, which I bought years ago to be protected of stupid modders and protect/help others, and then they stole it (still not sure why I bought it... idiot). Not buying another one though. Shouldn't have bought this one anyways. I'm stupid.

Anyways. Without it enabled, the syscall works. If I enabled it back, the PS3 freezes. The mod menu was time bombed (stolen from all who paid, basically, and maybe was good for stupid modders, but the good ones paid the price too). As of some date I don't remember, it would stop working. So we need to put back the date to 2018 or something to get it to work. It would work until some time. Now maybe SSL certificate check doesn't let the date be in 2018 anymore...

The mod menu is also powerful, as said by people and the developer himself. So either it's too much for the PS3 to handle, or the menu has some check on which if the date changes to later than the time to go down, it will freeze the PS3 so we can't EVER use the menu.

If it's the first option, no idea how to do it then. webMAN is the only one I know that can execute a syscall whenever people want. We just need to click on buttons or do it remotely. If it's the 2nd option... Wish I had decent reverse engineering knowledge by now and not just basic.... (Also ProDG knowledge and those programs for PS3 debugging, which would help straightaway), and I'd try to patch this thing (it seems to use packers, but could be a cool project for a begginer). Not even on Christmas I can go give stuff to people in a sleigh anymore..... Sad. Stupid modders and mod menu developers.

Thank you though. Was very helpful, and thanks for the amazing plugin! Really cool I can change the date whenever I want and all other things with syscalls. Interesting.

_**EDIT:**_ About what you said, cool to know then. If it installs without us needing to click on any buttons, I didn't know it would install if we didn't press anything (never tried, I think). Cool to know. Thanks again. Don't read it if it's too much haha. It's an explanation just out of curiosity, but no need to read it. Just wanted to write. Hours on this and not possible to get it back working.... Sad. Gonna get back to making an Android app. More productive.

**Comment by aldostools** [comment]
To set the time just assign the web command to the custom combo R2+SQUARE in /setup.ps3

**Comment by Edw590** [comment]
Yeah, I did that. Tested with combo and from the computer. Nothing changes. Always the same thing. Not webMAN's problem, I guess. Unless it's too much memory being used, but I'm not buying that one so easily. Must be the other option almost for sure. Thank you in any case though.

**Comment by Joonie86** [comment]
No further feedback
